### DISCUSSION TITLE
error with the exit function

### DISCUSSION THREAD

**Discussion by linebraks** [psvks/NotifyControl#11] [General]
im using synapse x  and this script

```lua
local Control = loadstring(game:HttpGetAsync("https://raw.githubusercontent.com/Obstronomic/NotifyControl/main/notifyControl.lua"))()

Control:exit()
```

![image](https://user-images.githubusercontent.com/113252011/189490310-5dbd090d-4339-4ad7-b5a8-544ff3f7bf09.png)

**Comment by OrEye** [comment]
`exit` is not on the table or statement indicated on `Control`

This might be the reasons why is happening this:

- You did a typo while scripting
- Statement not found.

You did a typo. The correct way is `Control:ExitClient()`

IF the problem resist contact or replay us again.  
THANKS

**Comment by psvks** [comment]
> im using synapse x and this script
> 
> ```lua
> local Control = loadstring(game:HttpGetAsync("https://raw.githubusercontent.com/Obstronomic/NotifyControl/main/notifyControl.lua"))()
> 
> Control:exit()
> ```
> 
> ![image](https://user-images.githubusercontent.com/113252011/189490310-5dbd090d-4339-4ad7-b5a8-544ff3f7bf09.png)

I did a typo in docs @OrEye. Sorry about that.

**Comment by linebraks** [comment]
@OrEye Okay thanks, now working
